golang/x/relay/client/cli/query.go

```diff
@@ -324,18 +325,36 @@ func GetCmdGetRequest(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
 		Use:     "getrequest <id>",
 		Example: "getrequest 12",
-		Long:    "Get a proof request using the associated ID",
+		Long:    "Get a proof request using the associated ID. ID can be an\n\"0x\" prepended hexbyte string or an integer",
 		Args:    cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cliCtx := context.NewCLIContext().WithCodec(cdc)
 
-			id, err := strconv.ParseUint(args[0], 10, 64)
-			if err != nil {
-				return err
+			var idBytes types.RequestID
+			if args[0][:2] == "0x" {
+				id, err := types.RequestIdFromHex(args[0])
+				if err != nil {
+					return err
+				}
+				idBytes = id
```

Comment:
this line seems uneccessary. I think you could just assign `idBytes` on line 335

Reply:
Is this what you mean?```var idBytes types.RequestID
			if args[0][:2] == "0x" {
				idBytes, err := types.RequestIdFromHex(args[0])
				if err != nil {
					return err
				}
				// idBytes = id
			} else {
				id, err := strconv.ParseUint(args[0], 10, 64)
				if err != nil {
					return err
				}

				// convert to bytes
				b := make([]byte, 8)
				binary.BigEndian.PutUint64(b, id)
				idBytes, err := types.NewRequestID(b)
				if err != nil {
					return err
				}
				// idBytes = newID
			}``` If I do that, it

Reply:
so because you're using `:=` in an inner scope, it's redclaring `idBytes` in that scope.  You can do it like this:
```
                        var idBytes types.RequestID
			var err error

			if args[0][:2] == "0x" {
				idBytes, err = types.RequestIdFromHex(args[0])
				if err != nil {
					return err
				}
			} else {
				id, err := strconv.ParseUint(args[0], 10, 64)
				if err != nil {
					return err
				}

				// convert to bytes
				b := make([]byte, 8)
				binary.BigEndian.PutUint64(b, id)
				idBytes, err = types.NewRequestID(b)
				if err != nil {
					return err
				}
			}```

Reply:
see this:
https://stackoverflow.com/a/38664932

Reply:
btw, this snipped is what I think should be a utility function :+1:

Reply:
ok, makes sense.  So we don't have a utils file?  Should I make one or put it somewhere else?